### PR TITLE
[BACKLOG-36890] Keyboard Navigation (A) for Layout

### PR DESCRIPTION
**Pull request by hv-vamsi**

### CODE REVIEW

File: impl/client/src/main/javascript/web/dojo/pentaho/common/propertiesPanel/Panel.js [hunk 1]
```diff
@@ -1288,6 +1288,7 @@ define([
 
       this.own(
         on(this.domNode, "contextmenu", lang.hitch(this, "onContextMenu")),
+		on(this.domNode, "keypress", lang.hitch(this, "onContextMenu")),
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/dojo/pentaho/common/propertiesPanel/Panel.js:1291]: Indentation is off.

**Comment by hv-vamsi** [comment]
@pentaho/wcag, Please review

**Review by dcleao — CHANGES_REQUESTED** [review]
All fine except formatting.

File: impl/client/src/main/javascript/web/dojo/pentaho/common/propertiesPanel/Panel.js (hunk 1 above)
Comment (hv-vamsi) [reply impl/client/src/main/javascript/web/dojo/pentaho/common/propertiesPanel/Panel.js]: addressed

**Review by phalguni-a — APPROVED** [review]

**Pull request merged**